release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -127,6 +127,12 @@ following notable technical changes:
 `generate crds` commands so that users can opt-in to `v1` CRDs. The default setting
 is `v1beta1`.
 
+Ansible-based Operator enhancements include:
+* Support for relative Ansible roles and playbooks paths in the Ansible-based Operator watches files.
```

Comment:
The term `watches` appears to be capitalized in other areas of the docs (e.g., [here](https://docs.openshift.com/container-platform/4.4/operators/operator_sdk/osdk-ansible.html#osdk-ansible-watches-file_osdk-ansible)).